root/static/scripts/release-editor/trackParser.js

```diff
@@ -33,12 +33,13 @@ releaseEditor.trackParser = {
     trackNumber: /^(?:M[\.\-])?([０-９0-9]+(?:-[０-９0-9]+)?)(?:\.|．|\s?-|:|：|;|,|，|$)?/,
     vinylNumber: /^([０-９0-9a-z]+)(?:\/[０-９0-9a-z]+)?(?:\.|．|\s?-|:|：|;|,|，|$)?/i,
 
-    trackTime: /\(?((?:[0-9０-９]+[：，．':,.])?[0-9０-９\?]+[：，．':,.][0-5０-５\?][0-9０-９\?])\)?$/,
+    trackTime: new RegExp("\\(?((?:[0-9０-９]+[：，．':,.])?[0-9０-９\\?]+[：，．'\\+:,.][0-5０-５\\?][0-9０-９\\?])\\)?$"),
```

Comment:
It is not about track time, it is about superseding `separators` (defined on [line 30](https://github.com/metabrainz/musicbrainz-server/pull/774/files#diff-95bd94154407c3fafde7a3dd2dfcd828R30)) with a custom delimiter.